### ISSUE TITLE
[AVM Module Issue]: runtime version cannot have default value

### ISSUE THREAD

**Issue by jgrexa** [Azure/terraform-azurerm-avm-res-web-site#22]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Module specific issue

- [X] I confirm that this issue is about this AVM module and not about the resource itself, or how to use Terraform.

### Issue Type?

Feature Request

### (Optional) Module Version

0.1.2

### Description

dotnet_version contains default value and when other runtime version is specified (for example `python_version=x.y`), terraform is failing with errors
```
│ "site_config.0.application_stack.0.java_version": only one of
│ `site_config.0.application_stack.0.docker,site_config.0.application_stack.0.dotnet_version,site_config.0.application_stack.0.java_version,site_config.0.application_stack.0.node_version,site_config.0.application_stack.0.powershell_core_version,site_config.0.application_stack.0.python_version,site_config.0.application_stack.0.use_custom_runtime`
│ can be specified, but `site_config.0.application_stack.0.dotnet_version,site_config.0.application_stack.0.python_version` were specified.
```

**Comment by donovm4** [comment]
addressing bug fix in v0.2.0

**Comment by donovm4** [comment]
fixed in v0.2.0
